webapp/platform/client/package.json

```diff
@@ -10,7 +10,8 @@
   "files": [
     "lib"
   ],
-  "main": "./lib/index.js",
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
```

Comment:
Specifying this lets TS find the type definitions for `@mattermost/client` when it's used as a node module. I actually knew that this field existed, so I'm not sure how I forgot it here